### PR TITLE
Rename test/functional dir to test/unit

### PR DESCRIPTION
**Pull request by estherkim**
To avoid confusion when we add e2e/functional tests

### CODE REVIEW

**Comment by lannka** [comment]
thanks @estherkim . could you explain what is `e2e/functional`? can you give an example?

**Comment by estherkim** [comment]
Sure, an e2e/functional test is written from the perspective of the user, that usually involves interaction with the UI. An example functional test would be like - 
```
visit website
find search input element
type in search text
type enter key
find result element
expect result element text to contain search text
```

**Comment by danielrozenberg** [comment]
There's a conflicting file, you'll need to rebase this and fix the issue

**Comment by estherkim** [comment]
Done! :)

File: ads/README.md [hunk 1]
```diff
@@ -343,7 +343,7 @@ gulp test --watch --nobuild --files=test/functional/{test-ads-config.js,test-int
```
Comment (rsimha) [inline ads/README.md]: I think you missed one on line 342.

Comment (estherkim) [reply ads/README.md]: Great catch - created a new PR below.

**Pull request merged**